index.html

```diff
@@ -127,6 +127,14 @@ <h6 class="header3">About Myself</h6>
                 </nav>
                  
             </section>
+            <footer class="footer">
+                <h2 class="contact">Contact me</h2>
+                <p class="info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <input type="text" name="name" maxlength="30" placeholder="Iptisam Dawood">
+                <input type="email" name="email" placeholder="mukadamiptisam@gmail.com">
+                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+                <button class="btn" type="submit">Get it Touch</button>
+            </footer>
```

Comment:
- [ ] There should be no white space on the sides of your form, kindly remove it 
![image](https://user-images.githubusercontent.com/62705947/186419201-8cdd1749-cf73-4d4b-91d4-6849581e7be9.png)